apps/web/src/pages/templates/workflow/digest/WillBeSentHeader.tsx

```diff
@@ -0,0 +1,175 @@
+import { useCallback, useEffect } from 'react';
+import { useFormContext } from 'react-hook-form';
+import * as capitalize from 'lodash.capitalize';
+import { useMantineColorScheme } from '@mantine/core';
+import { DigestTypeEnum, DigestUnitEnum, MonthlyTypeEnum } from '@novu/shared';
+
+import { colors } from '../../../../design-system';
+import { pluralizeTime } from '../../../../utils';
+
+const getOrdinalValueLabel = (value: string) => {
+  if (value === 'day' || value === 'weekday') {
+    return value;
+  }
+  if (value === 'weekend') {
+    return 'weekend day';
+  }
+
+  return capitalize(value);
+};
+
+const getOrdinal = (num: string | number) => {
+  if (typeof num === 'string') {
+    const res = parseInt(num, 10);
+
+    if (isNaN(res)) {
```

Comment:
```suggestion
    if (Number.isNaN(res)) {
```
Is better to use the `Number` API implementation for `isNaN`.